dolphinscheduler-e2e/dolphinscheduler-e2e-core/src/main/java/org/apache/dolphinscheduler/e2e/core/DolphinSchedulerExtension.java

```diff
@@ -159,6 +151,39 @@ public Statement apply(Statement base, Description description) {
         rootPath = "/dolphinscheduler/ui/";
     }
 
+    private void getBrowserContainerByOsName() {
+        if (LOCAL_MODE && M1_CHIP_FLAG) {
+            imageName = DockerImageName.parse("seleniarm/standalone-chromium:4.1.2-20220227")
```

Comment:
If this image also works for non-M1, I'd just upgrade the image for all systems, not just M1

Reply:
This image only work for ARM64 chip.